packages/kbn-gateway/src/config/read_argv.ts

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+/**
+ * Borrowed from @kbn/apm-config-loader.
+ */
```

Comment:
This file (and corresponding tests) were simply copy-pasted from `@kbn/apm-config-loader` -- I didn't make any changes.